### DISCUSSION TITLE
Transfer learning on the 3D classification example

### DISCUSSION THREAD

**Discussion by addcue** [Project-MONAI/MONAI#6545] [General]
In the example for 3D classifications (https://github.com/Project-MONAI/tutorials/blob/main/3d_classification/ignite/densenet_training_array.py), we import a DenseNet-121 model with 3 spatial dimensions. How does this model become 3D? Aren't the original ImageNet images 2D?

**Comment by KumoLiu** [comment]
Hi @addcue, you can set the `spatial_dims` to 2 as well, and yes the original ImageNet images are 2D, but the default `pretrained` arg in the network is `False`. You can set it to `True` with `spatial_dims` equal to 3.
https://github.com/Project-MONAI/MONAI/blob/960249f974fdce9c86b27f67cd5afc9ab918eb7a/monai/networks/nets/densenet.py#L173
https://github.com/Project-MONAI/MONAI/blob/960249f974fdce9c86b27f67cd5afc9ab918eb7a/monai/networks/nets/densenet.py#L304
Hope it can help you, thanks!

**Reply by addcue** [reply]
Yes, I was wondering how that works though. Since the model was trained on 2D images from ImageNet, how can we just change the model to 3D? And what exactly changes about the model?

**Reply by KumoLiu** [reply]
Hi @addcue, I think we can use 3d Convolution and Pooling layer, right?

**Reply by addcue** [reply]
But we can retain the weights from the 2D model?

**Reply by KumoLiu** [reply]
You can only set `pretrained=True` when `spatial_dims=2`.
